YukkiMusic/utils/decorators/admins.py

```diff
@@ -148,26 +122,25 @@ async def wrapper(client, CallbackQuery):
                     CallbackQuery.message.chat.id,
                     CallbackQuery.from_user.id,
                 )
-            except:
+            except Exception:
                 return await CallbackQuery.answer(
                     _["general_5"], show_alert=True
                 )
-            if not a.can_manage_voice_chats:
-                if CallbackQuery.from_user.id not in SUDOERS:
-                    token = await int_to_alpha(
-                        CallbackQuery.from_user.id
-                    )
-                    _check = await get_authuser_names(
-                        CallbackQuery.from_user.id
-                    )
-                    if token not in _check:
-                        try:
-                            return await CallbackQuery.answer(
-                                _["general_5"],
-                                show_alert=True,
-                            )
-                        except:
-                            return
+            if not a.can_manage_video_chats and CallbackQuery.from_user.id not in SUDOERS:
```

Comment:
It looks like the `ChatMember` type doesn't have the `can_manage_voice_chats` field.

Pyrogram v2.0 update information says:

> Renamed the field can_manage_voice_chats to can_manage_video_chats in the class [ChatPrivileges](https://docs.pyrogram.org/api/types/ChatPrivileges.html#pyrogram.types.ChatPrivileges).

Privileges should be looked at in `ChatMember.privileges` in this and all other places.

Reply:
yeah. u right

`can_manage_video_chats (bool, optional) – Groups and supergroups only. True, if the administrator can manage video chats (also called group calls).`